README.md

```diff
@@ -40,7 +40,11 @@ pip install volttron
     > vctl shutdown --platform
     ```
 
-Full VOLTTRON documentation available at [VOLTTRON Readthedocs](https://volttron.readthedocs.io)
+## Docker
```

Comment:
We will move it shortly.  I was originally thinking that we would install volttron within it...but since we are shelling around it then we can move it out.